include/openssl/hmac.h

```diff
@@ -203,4 +270,12 @@ BSSL_NAMESPACE_END
 
 #endif
 
+
+// Errors
+
+#define HMAC_R_MISSING_PARAMETERS 100
+#define HMAC_R_BUFFER_TOO_SMALL 102
+#define HMAC_R_SET_PRECOMPUTED_KEY_EXPORT_NOT_CALLED 103
+#define HMAC_R_NOT_CALLED_JUST_AFTER_INIT 104
```

Comment:
It's not documented well, but I think you will want to run `go run ./util/make_errors.go hmac` from the root of the repository after adding these. This will generate a `crypto/err/hmac.errordata` file that will be visible if you do a `git status`. It may also reformat the hmac.h header here as well.

You will then want to add that file to the list [here](https://github.com/aws/aws-lc/blob/main/crypto/CMakeLists.txt#L285).

Reply:
Thanks! Done.